profile_bluesky/startup/instrument/devices/lakeshores.py

```diff
@@ -0,0 +1,284 @@
+"""i
+Lakeshore temperature controllers
+"""
+
+__all__ = ['lakeshore_336','lakeshore_340lt','lakeshore_340ht']
+
+from instrument.session_logs import logger
+logger.info(__file__)
+
+from apstools.synApps.asyn import AsynRecord
+from ophyd import Component, Device, Signal
+from ophyd import EpicsSignal, EpicsSignalRO, EpicsSignalWithRBV
+from ophyd import FormattedComponent,PVPositioner
+from ophyd import Kind
+
+# TODO: fixes bug in apstools/synApps/asyn.py
+class MyAsynRecord(AsynRecord):
+    binary_output_maxlength = Component(EpicsSignal, ".OMAX")
```

Comment:
Now fixed in [apstools](https://github.com/BCDA-APS/apstools/commit/480c1dd00a5a9f0da89e79db921b68fb652c9e08#diff-61d3347261a897120428dc0b9401c4a6) but name changed to `output_maxlength` since it applies to both binary and ASCII data streams.

For legacy support, `binary_output_maxlength` is (now) an alias for `output_maxlength` but that could be dropped in a future release.

Reply:
Use `AsynRecord` from `apstools` now.  Delete `MyAsynRecord` and change below.